processor/resourcedetectionprocessor/internal/aws/eks/metadata.yaml

```diff
@@ -10,4 +10,8 @@ resource_attributes:
   cloud.platform:
     description: The cloud.platform
     type: string
+    enabled: true
+  k8s.cluster.name:
+    description: The EKS cluster name. This attribute is currently only available when running on EC2 instances, and requires permission to run the EC2:DescribeInstances action.
+    type: string
     enabled: true
```

Comment:
This will be enabled by default, let me know if it should be disabled by default.

Reply:
Users might have other processor adding this attribute, and since there is no official cluster name concept in k8s itself, they could've set it to some variation of eks name. I think it would be safer to leave it disabled initially.